stellarphot/settings/models.py

```diff
@@ -545,27 +564,81 @@ class PhotometryOptions(BaseModelWithTableRep):
     fwhm_by_fit: bool = True
 
 
+class PassbandMapEntry(BaseModel):
+    """
+    long loing liong liong liong long long
+    multiline name
+
+    Parameters
+    ----------
+    your_filter_name : str
+        Instrumental filter name.
+
+    aavso_filter_name : `stellarphot.settings.AAVSOFilters`
+        AAVSO filter name.
+
+    """
+
+    your_filter_name: str
+    aavso_filter_name: Annotated[AAVSOFilters, Field(title="AAVSO Filter Name")]
+
+
 class PassbandMap(BaseModelWithTableRep):
     """
     Class to represent a mapping from one set of filter names to another.
 
     Parameters
     ----------
-    yours_to_aavso : dict[str, str]
-        A dictionary containing instrumental passband names as keys and
-        AAVSO passband names as values. This is used to rename the passband
-        entries in the output photometry table.
+    your_filter_names_to_aavso : list[`stellarphot.settings.PassbandMapEntry`]
```

Comment:
OK, this makes sense, although again, not seeing how `stellarphot/core.py` knows about this?